content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-linux.md

```diff
@@ -0,0 +1,17 @@
+# Update Automatically on Linux
+
+On Linux, you can set up `rippled` to automatically upgrade to the latest version with a one-time `cron` configuration. Ripple recommends enabling automatic updates if possible.
+
+These instructions assume you have already installed `rippled` [from the `yum` repository (CentOS/RedHat)](install-rippled-on-centos-rhel-with-yum.html) or [using `apt` (Ubuntu/Debian)](install-rippled-on-ubuntu.html).
+
+**Warning:** As of version 1.3.0, the install and upgrade process has changed. Automatic updates **will not** upgrade from `rippled` 1.2.4 or lower to `rippled` 1.3.0 or higher. If you have `rippled` v1.2.4 or lower installed, follow the [rippled 1.3.0 migration instructions](rippled-1-3-0-migration-instructions.html) first. After performing the migration, automatic updates should work again.
```

Comment:
I think this is not strictly true since we are planning on pushing a one-time (and final) update to the ripple-repo package that switches existing auto-update users over to the new repositories and thus they will get the 1.3 update. That said, the manual upgrade steps you've outlined makes things cleaner and allows the users to remove the ripple-repo package.

Reply:
I'll have to dig more into how that will work for users using the existing auto-update script...